glfs.c

```diff
@@ -634,6 +635,41 @@ static int tcmu_glfs_write(struct tcmu_device *dev,
 	return SAM_STAT_TASK_SET_FULL;
 }
 
+static int tcmu_glfs_get_image_size(struct tcmu_device *dev,
+                                    uint64_t new_size)
+{
+	struct glfs_state *gfsp = tcmu_get_dev_private(dev);
+	struct stat st;
+	int ret;
+
+	ret = glfs_lstat(gfsp->fs, gfsp->hosts->path, &st);
+	if (ret) {
+		tcmu_dev_err(dev, "glfs_lstat failed: %m\n");
+		return ret;
+	}
+
+	if (st.st_size != new_size) {
+		tcmu_dev_err(dev, "Mismatched sizes. glfs image size %lld. Requested new size %" PRIu64 ".\n",
```

Comment:
```
In file included from /home/XXX/tcmu-runner/tcmu-runner.h:33:0,
                 from /home/XXX/tcmu-runner/glfs.c:32:
/home/XXX/tcmu-runner/glfs.c: In function ‘tcmu_glfs_get_image_size’:
/home/XXX/tcmu-runner/glfs.c:652:84: error: expected ‘)’ before ‘PRIu64’
   tcmu_dev_err(dev, "Mismatched sizes. glfs image size %lld. Requested new size %" PRIu64 ".\n",
                                                                                    ^
/home/XXX/tcmu-runner/libtcmu_log.h:71:76: note: in definition of macro ‘tcmu_dev_err’
 #define tcmu_dev_err(dev, ...)  {tcmu_err_message(dev, __func__, __LINE__, __VA_ARGS__);}
```

Reply:
... missing ```#include <inttypes.h>```

Reply:
Thanks Jason.

I fixed this for you @pkalever.